whole-repo-tests/test_security.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2018 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+from hypothesistooling.scripts import pip_tool
+from hypothesistooling.projects.hypothesispython import PYTHON_SRC
+
+
+def test_bandit_passes_on_hypothesis():
+    # pypi.org/project/bandit has the table of error codes, or `bandit --help`
+    pip_tool('bandit', '--skip', 'B101,B102,B110,B303,B311',
```

Comment:
Why can't you specify these by their names. 😭

Reply:
PYQCA doesn't like names :sob: